internal/checker/preprocessor.go

```diff
@@ -7,9 +7,13 @@ import (
 	"unicode"
 )
 
-func processScript(script string, ruleConf *RuleConf) string {
+const rulesetDirConstantName = "@ruleset"
+
+func processScript(script string, ruleConf *RuleConf, rulesetPath string) string {
 	newScript, decorators := getDecorators(script)
 
+	newScript = strings.ReplaceAll(newScript, rulesetDirConstantName,  rulesetPath)
```

Comment:
I don't think it is good idea to replace all `@ruleset` matches in script
I believe we should do this kind of logic internally in `api.fs` code

Reply:
We should discuss this one with @cooligus